<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicPlaybackControls</class>
 <widget class="QWidget" name="musicPlaybackControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="videoWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="albumLabel">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="prevBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="lyricLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>            歌词在这里</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="currentTime">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>00:00</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="processBar">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="totalTime">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>04:21</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="switchModeBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="volumeBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="lyricBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="enjoyBtn">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
